rllib/utils/replay_buffers/prioritized_episode_replay_buffer.py

```diff
@@ -215,7 +216,8 @@ def sample(
                 be the one executed n steps before such that we always have the
                 state-action pair that triggered the rewards.
                 If `n_step` is a tuple, it is considered as a range to sample
-                from. If `None`, we sample from the range (1, 5).
+                from. If `None`, we sample from the range (1, 5). note, sampling
```

Comment:
Dumd question: Should the default for n-step not simply be 1, instead of (1, 5)?

Reply:
On the above explanation for the returned tuples:

Why do we do this (imo unintuitively) looking back, meaning we explain here that in "new_obs", we store the "obs" and in "obs" we store the obs from n timesteps back. Shouldn't it be the other way around? In "obs", we store the "obs" and in "new_obs", we store the obs from n timesteps into the future. This would - in my understanding - be the normal way to describe a n-tuple returned from a replay buffer for Q-learning algos.

Reply:
Great question - I expected this one to come. I followed here the paper that introduced Prioritized Experience Replay. In __Algorithm 1__ the authors sample `j` and then get `j-1` for it. They then also update the priority for `j`.

Reply:
In regard to the default: you proposed earlier that when `None` we should sample from 1 to 5. I will change this to 1 now.